curriculum/challenges/english/14-responsive-web-design-22/learn-basic-css-by-building-a-cafe-menu/5f356ed63e0fa262326eef05.md

```diff
@@ -7,11 +7,11 @@ dashedName: step-24
 
 # --description--
 
-Now make the background color of the `div` element to be `burlywood`.
+Now make the background color of the `div` element to be `burlywood` using the existing `div` selector.
 
 # --hints--
 
-You should set the `background-color` property to `burlywood`.
+You should set the `background-color` property in the existing `div` selector to `burlywood`.
```

Comment:
I agree with @moT01 that the briefer wording he suggested is preferable here

Reply:
I agree!
```suggestion
In the existing `div` selector, change the background color to `burlywood`.
```

Reply:
I wasn't clear - but my suggestion was for the instructions, not the hint.